pyEpiabm/pyEpiabm/core/place.py

```diff
@@ -33,10 +35,14 @@ def __init__(self, loc: typing.Tuple[float, float],
         """
         self._location = loc
         self.persons = []
+        self.person_groups = {0: []}
+        self.num_person_groups = 0
         self.place_type = place_type
         self.max_capacity = 50
+        self.mean_capacity = 25
```

Comment:
Can this be specified in params?

Reply:
Was asking Nick if we could set some function on the placetype enum to default parameters for each enum, but by that point probably just want to be reading in stuff. Possibly want all the parameters to be a list length 5 so might need some indexing

Reply:
Yeah I think reading in stuff is the way to go - at the moment I think we're sticking everything we'll want to read in in the parameters file (so it could be theoretically configurable from there), but will move that to a `.json` or similar soon. Given the rigis enum setup I think lists are fine - just worth making sure we have the place types we want (i.e. no schools for example) before we commit to that!

Reply:
Put my parameters in as a dictionary anyway